cmd/soci/commands/create.go

```diff
@@ -32,6 +32,7 @@ const (
 	buildToolIdentifier = "AWS SOCI CLI v0.1"
 	spanSizeFlag        = "span-size"
 	minLayerSizeFlag    = "min-layer-size"
+	xattrFlag           = "xattr-optimization"
```

Comment:
I imagine we will eventually have more than one of this sort of opt-in optimization available. Instead of a top level flag for each, perhaps something like `--optimizations xattr,foo,bar`?

Reply:
Do we want this to be configurable long term? As I understand it, we don't have total confidence that this is safe to do (we think it is, but haven't rigorously tested) and so we shouldn't do it by default. I think we should put it behind a flag that makes it more clear that it might go away in the future like `experimental-xattr-optimization`.

If we decide to keep it optional in the future, we can design the actual flag name.

Reply:
I think `--disable-xattrs` makes sense - we can then talk about how the optimizations in the description.

Reply:
I predict that, long term, we will have *some* optional optimizations. My proposal here is meant to accommodate that, regardless of what they end up being, without us repeatedly changing our command line flag list.